instrumentation/instabeego/web_server_handler.go

```diff
@@ -0,0 +1,35 @@
+// (c) Copyright IBM Corp. 2023
+
+//go:build go1.18
+// +build go1.18
+
+// Package instabeego provides Instana instrumentation for the beego framework.
+package instabeego
+
+import (
+	"net/http"
+
+	beego "github.com/beego/beego/v2/server/web"
+	beecontext "github.com/beego/beego/v2/server/web/context"
+	instana "github.com/instana/go-sensor"
+)
+
+// InstrumentWebServer wraps beego's handlers execution. Adds tracing context and handles entry span.
+// It should be added as a first Middleware to the beego, before defining handlers.
+func InstrumentWebServer(sensor *instana.Sensor) {
```

Comment:
Again, please use the instana.TracerLogger interface. 
You can replace the concrete instana.Sensor struct everywhere, so I don't bother you in every comment haha

Reply:
done